apps/astarte_data_updater_plant/lib/astarte_data_updater_plant/data_updater/endpoint.ex

```diff
@@ -0,0 +1,43 @@
+#
+# This file is part of Astarte.
+#
+# Copyright 2025 SECO Mind Srl
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+defmodule Astarte.DataUpdaterPlant.DataUpdater.Endpoint do
```

Comment:
We can evaluate whether it makes sense to use Ecto types from Astarte core, as we're doing in other microservices, for example to simplify parsing of database integers into enums/atoms.

https://github.com/astarte-platform/astarte/blob/60d7364826d5cdb0a470ce9a0f8fb16444621b09/apps/astarte_realm_management/lib/astarte_realm_management/realms/endpoint.ex

Reply:
Good catch!